mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubClientProcess.kt

```diff
@@ -142,4 +129,30 @@ public class GitHubClientProcess :
     internal fun inject(gitHubClientService: GitHubClientService) {
         this.gitHubClientService = gitHubClientService
     }
+
+    private companion object {
+        /**
+         * Determines and returns the time from which mentions should be updated.
+         *
+         * If updates were already performed, it returns the time of the last successfully
+         * completed update. If no updates are made, it returns the time equal to midnight
+         * of the previous work day.
+         */
+        private fun determineFromWhenFetchMentions(whenLastSuccessfulUpdate: Timestamp): Timestamp {
```

Comment:
Also, for the future. When you write the code with the terms this long, most likely you are doing something wrong:
```
determineFromWhenFetchMentions(whenLastSuccessfulUpdate: Timestamp)
```
People get tired just trying to split this whole idea into meaningful parts. It's just an over-use of camelCasing as an idea.